mne_bids/path.py

```diff
@@ -52,15 +54,17 @@ class BIDSPath(object):
         The recording name for this item. Corresponds to "rec".
     space : str | None
         The coordinate space for an anatomical file. Corresponds to "space".
-    prefix : str | None
-        The prefix for the filename to be created. E.g., a path to the folder
+    root : str | None
+        The directory for the filename. E.g., a path to the folder
         in which you wish to create a file with this name.
-    suffix : str | None
-        The suffix for the filename to be created. E.g., 'audio.wav'.
+    kind : str | None
+        The suffix for the filename. E.g., 'ieeg'.
```

Comment:
"suffix", here, is too ambiguous. Who not state what it is: "The recording kind. Will be be appended to the filename before the extension.